extension.html

```diff
@@ -0,0 +1,119 @@
+      </p>
+    </section>
+    <section dfn-for="extensions-to-pointerevents-interface">
+      <h2>Extensions to the <code>PointerEvent</code> interface</h2>
+      <p>
+        The following section describes extensions to the existing <a href="https://w3c.github.io/pointerevents/index.html#pointerevent-interface"><code>PointerEvent</code> interface</a>.
+      </p>
+      <div>
+          <pre class="idl">
+partial dictionary PointerEventInit {
+    sequence&lt;PointerEventInit> coalescedEventInits = [];
+};
+
+partial interface PointerEvent {
+    sequence&lt;PointerEvent> getCoalescedEvents();
+};
+          </pre>
+          <dl dfn-for="PointerEvent" link-for="PointerEvent">
+              <dt><dfn>getCoalescedEvents</dfn></dt>
+              <dd>
+                  <p> Returns a sequence of all <code>PointerEvents</code> that were coalesced into the received event. The following attributes of the coalesced events will always have the same value as the received event: <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, and particularly <code>target</code>. But other attributes might be different. The events in the sequence will have increasing <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a> ([[!WHATWG-DOM]]). So the first event will have the smallest <code>timeStamp</code>. Also the received event's <code>timeStamp</code> will be greater than or equal to the last event's <code>timeStamp</code> in the sequence. None of the events in the sequence will have coalesced events. When an event has no coalesed event, this function returns an empty sequence.</p>
+              </dd>
+          </dl>
+          <p>The <code>PointerEventInit</code> dictionary is used by the <code>PointerEvent</code> interface's constructor to provide a mechanism by which to construct untrusted (synthetic) pointer events. It inherits from the <code>MouseEventInit</code> dictionary defined in [[!DOM-LEVEL-3-EVENTS]]. The steps for constructing an event are defined in [[!DOM4]]. The <coalescedEventInits> will be used to create coalesced PointerEvents of the generated untrusted synthetic pointer event. The number of coalesced events will be the same as <code>coalescedEventInits</code> list and all of their attributes will be have the same values as the generated event unstrusted synthetic pointer event except those that have a different value in their coalescedEventInits entry. See the <a href="#examples" title="examples">examples</a> for sample code demonstrating how to fire an untrusted pointer event.</p>
+          </p>
+      </div>
+    </section>
+
```

Comment:
This makes sense, and is an extra developer complexity I didn't appreciate.  Android and iOS don't have this problem since they bundle the pointers together.  At least I think it would be very rare that an app would want to BOTH reason about the relationship between different fingers AND look at historical events (maybe a drawing app that uses the distance between fingers to influence the stroke width or something).